.github/workflows/ci.yaml

```diff
@@ -58,6 +58,9 @@ jobs:
       - name: Build 🔧
         env:
           PR_NUMBER: ${{ github.event.pull_request.number }}
+          GATSBY_ALGOLIA_APP_ID: ${{ secrets.DOCS_ALGOLIA_APP_ID }}
+          ALGOLIA_ADMIN_KEY: ${{ secrets.DOCS_ALGOLIA_ADMIN_KEY }}
```

Comment:
We can try with the write key but the docs of `gatsby-plugin-algolia` specify to use the admin key, I think that's because they pull settings from the dashboard

Reply:
@sileht Updated with `ALGOLIA_WRITE_KEY `